File: sdkwork-backend-sdk-typescript/src/types/plus-vip-point-change-vo.ts
/** VIPç§¯ååå¨è®°å½VO */
export interface PlusVipPointChangeVO {
  /** åå»ºæ¶é´ */
  createdAt?: string;
  /** æ´æ°æ¶é´ */
  updatedAt?: string;
  /** ç¨æ·IDï¼å³èplus_userè¡¨ */
  userId?: number;
  /** åå¨ç±»å */
  changeType?: 'DEFAULT' | 'RECHARGE' | 'CONSUME' | 'REWARD' | 'ADJUST' | 'EXPIRE';
  /** åå¨æ°é(æ­£æ°ä¸ºå¢å ï¼è´æ°ä¸ºåå°) */
  changeAmount?: number;
  /** åå¨åä½é¢ */
  beforeBalance?: number;
  /** åå¨åä½é¢ */
  afterBalance?: number;
  /** ä¸å¡æ¥æºID(å¦åå¼è®°å½IDãè®¢åIDç­) */
  sourceId?: number;
  /** ä¸å¡æ¥æºç±»å */
  sourceType?: 'DEFAULT' | 'PURCHASE' | 'DAILY_REWARD' | 'COUPON_POINTS_EXCHANGE' | 'COUPON_POINTS_ROLLBACK' | 'SYSTEM' | 'OTHER';
  /** å¤æ³¨ä¿¡æ¯ */
  remark?: string;
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/PlusVipPointChangeVO.java
package com.sdkwork.backend.model;

public class PlusVipPointChangeVO {
    private String createdAt;
    private String updatedAt;
    private Integer userId;
    private String changeType;
    private Integer changeAmount;
    private Integer beforeBalance;
    private Integer afterBalance;
    private Integer sourceId;
    private String sourceType;
    private String remark;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public Integer getUserId() {
        return this.userId;
    }
    
    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public String getChangeType() {
        return this.changeType;
    }
    
    public void setChangeType(String changeType) {
        this.changeType = changeType;
    }

    public Integer getChangeAmount() {
        return this.changeAmount;
    }
    
    public void setChangeAmount(Integer changeAmount) {
        this.changeAmount = changeAmount;
    }

    public Integer getBeforeBalance() {
        return this.beforeBalance;
    }
    
    public void setBeforeBalance(Integer beforeBalance) {
        this.beforeBalance = beforeBalance;
    }

    public Integer getAfterBalance() {
        return this.afterBalance;
    }
    
    public void setAfterBalance(Integer afterBalance) {
        this.afterBalance = afterBalance;
    }

    public Integer getSourceId() {
        return this.sourceId;
    }
    
    public void setSourceId(Integer sourceId) {
        this.sourceId = sourceId;
    }

    public String getSourceType() {
        return this.sourceType;
    }
    
    public void setSourceType(String sourceType) {
        this.sourceType = sourceType;
    }

    public String getRemark() {
        return this.remark;
    }
    
    public void setRemark(String remark) {
        this.remark = remark;
    }
}
